### DISCUSSION TITLE
Issue with Copy and Paste

### DISCUSSION THREAD

**Discussion by ldelossa** [kovidgoyal/kitty#6874] [Q&A] [answered]
I'm facing a rather odd issue ever since I updated to Fedora 39. 

I'm very used to double clicking text to get it into my system clipboard, however this no longer works, unless I perform a copy somewhere else in another application. Once I do this with "ctrl-c" in a browser, for example, I can continue to double click on text in the terminal and it moves to my system clipboard. 

Any idea what this may be? I can't pin point on when it exactly happens, but seems to reoccur every so often when I come back to my computer.

**Comment by kovidgoyal** [comment, marked as answer]
This will be a wayland issue. Set linux_display_backend x11 in
kitty.conf and you will be fine.

**Reply by ldelossa** [reply]
Thanks going to try this

**Reply by ldelossa** [reply]
looks good, thanks a lot.

Am I no longer using wayland backend by doing this? Would I want to change this setting back in some time?
